File: pertemuan 5/PredikatKelulusan.java
import java.util.Scanner;

public class PredikatKelulusan {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        String nDepan, nBelakang, nim, pStudi, fakultas;
        double ipk;
        int tahunMasuk, tahunLulus;

        System.out.print("Masukan nama depan Anda = ");
        nDepan=scan.nextLine();
        System.out.print("Masukan nama belakang Anda = ");
        nBelakang=scan.nextLine();
        System.out.print("Masukan nim Anda = ");
        nim=scan.nextLine();
        System.out.print("Masukan program studi Anda = ");
        pStudi=scan.nextLine();
        System.out.print("Masukan fakultas Anda = ");
        fakultas=scan.nextLine();
        System.out.print("Masukan ipk Anda = ");
        ipk=scan.nextDouble();
        System.out.print("Masukan tahun masuk Anda = ");
        tahunMasuk=scan.nextInt();
        System.out.print("Masukan tahun lulus Anda = ");
        tahunLulus=scan.nextInt();

        System.out.println("Nama Lengkap = " + nDepan + nBelakang);
        System.out.println("NIM = " + nim);
        System.out.println("Program Studi = " + pStudi);
        System.out.println("Fakultas = " + fakultas);
        System.out.println("IPK =" + ipk);
        
        int lamaStudi = tahunLulus - tahunMasuk + 1;
        if (ipk >= 3.51 && ipk <= 4 && lamaStudi <= 4) {
            System.out.println("Predikat Lulus = Dengan Pujian (Summa CumLaude)");
        } else if (ipk >= 3.51 && ipk <= 4) {
            System.out.println("Predikat Lulus = Dengan Pujian (CumLaude)");
        } else if (ipk >= 3.01 && ipk < 3.51) {
            System.out.println("Predikat Lulus = Sangat Memuaskan");
        } else if (ipk >= 2.76 && ipk < 3.01) {
            System.out.println("Predikat Lulus = Memuaskan");
        } else if (ipk >= 2.00 && ipk < 2.76) {
            System.out.println("Predikat Lulus = Cukup");
        } else if (ipk < 2.00) {
            System.out.println("Predikat Lulus = Tidak Lulus");
        } else if (ipk > 4 || ipk < 0) {
            System.out.println("Predikat Lulus = Tidak Valid");
        }
    }
}

File: pertemuan 5/predikatkelulusan.py
nama = str(input("Masukan Nama Anda = "))
nim = str(input("Masukan NIM Anda = "))
pstudi = str(input("Masukan Program Studi Anda = "))
fakultas = str(input("Masukan Fakultas Anda = "))
ipk = str(input("Masukan IPK Anda = " ))
studi = str(input("Masukan Lama Studi Anda = "))

print("Berikut Data Diri Anda")
print("Nama Anda adalah = " + nama)
print("NIM Anda = " + nim)
print("Jurusan Anda = " + pstudi)
print("Fakultas = " + fakultas)
print("Dengan IPK = " + ipk)
print("Lama Studi = " + studi)

if (float(ipk) >= 3.51 and float(ipk) <= 4 and float(studi) <= 4): 
    print("Predikat Lulus Anda Adalah = Dengan Pujian (Summa CumLaude)")
elif (float(ipk) >= 3.51 and float(ipk) <= 4): 
    print("Predikat Lulus Anda Adalah = Dengan Pujian (CumLaude)")
elif (float(ipk) >= 3.01 and float(ipk) < 3.51): 
    print("Predikat Lulus Anda Adalah = Sangat Memuaskan")
elif (float(ipk) >= 2.76 and float(ipk) < 3.01): 
    print("Predikat Lulus Anda Adalah = Memuaskan")
elif (float(ipk) >= 2.00 and float(ipk) < 2.76): 
    print("Predikat Lulus Anda Adalah = Cukup")
elif (float(ipk) < 2.00): 
    print("Predikat Lulus Anda Adalah = Tidak Lulus")
elif (float(ipk) > 4.00 or float(ipk) < 0): 
    print("Predikat Lulus Anda = Tidak Valid")






